.travis.yml

```diff
@@ -29,13 +29,27 @@ jobs:
       install:
         - python3 codalab_service.py build services --version ${TRAVIS_BRANCH} --pull $([ -z "${CODALAB_DOCKER_USERNAME}" ] || [ "$TRAVIS_PULL_REQUEST_BRANCH" != "" ] || echo "--push")
         - python3 codalab_service.py start --services default monitor --version ${TRAVIS_BRANCH}
+        - pip install -r requirements.txt
+      before_script:
+        # For testing CodaLab in Chrome
```

Comment:
should we add these scripts to a separate file (such as `test-setup.sh`) so that it's easy to set up and run Selenium tests locally?

Reply:
That's a good idea. I can put it in a separate script.